utils/tasks_library.py

```diff
@@ -0,0 +1,52 @@
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+from dissertation.models.offer_proposition import OfferProposition
+
+YEAR = 1
+
+
+def offer_proposition_extend_dates():
+    all_offer_propositions = OfferProposition.objects.all()
+    logs = ''
+    for offer_proposition in all_offer_propositions:
+        logs += check_dates_of_offer_proposition(offer_proposition)
+    if logs == '':
```

Comment:
`if not logs`est plus pythonesque.

Reply:
Voilà j'ai fait quelques changements ;-)